python-lib/recipe_config_loading.py

```diff
@@ -19,7 +19,7 @@ def check_and_get_groupby_columns(recipe_config, dataset_columns):
 
 
 def _format_groupby_columns(recipe_config):
-    if recipe_config.get('advanced_activated') and recipe_config.get('groupby_column'):
+    if recipe_config.get('advanced_activated') and recipe_config.get('groupby_column') and recipe_config.get('groupby_columns') is None:
```

Comment:
'groupby_columns' is an empty list when not filled

Reply:
replaced in https://github.com/dataiku/dss-plugin-timeseries-preparation/pull/27/commits/2965932e68cc1b45a31128bfdfac4df62b0754de